index.php

```diff
@@ -1,6 +1,8 @@
 <?php
 include('loklak.php');
-
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
```

Comment:
I can remove it. As it should not be present during production. 
Developers can add it as per there requirement. 
This can be handled later using config file. 
@sudheesh001